test/unit/contract_spec.ts

```diff
@@ -0,0 +1,277 @@
+import { xdr, ContractSpec, Address } from "../../lib";
+//@ts-ignore
+import spec from "../spec.json";
+import { expect } from "chai";
+
+const publicKey = "GCBVOLOM32I7OD5TWZQCIXCXML3TK56MDY7ZMTAILIBQHHKPCVU42XYW";
+const addr = Address.fromString(publicKey);
+
+describe("Can parse base64 entries", function () {
+  const SPEC = new ContractSpec(spec);
```

Comment:
this should be in an `it`, I think, to catch errors? like so:

```suggestion
let SPEC;

it('constructs from base64 entries', function() {
  SPEC = new ContractSpec(spec);
}

it('throws with no spec', function() {
  expect(() => new ContractSpec([])).to.throw(/expected error message here/i);
}
```

Reply:
whereas the `describe` should be something more like "build values using ContractSpec"